main.js

```diff
@@ -47,18 +47,45 @@ function validator(emailAcquire) {
   return false;
 }
 
+// Portfolio: preserve data in the browser
+const nameData = document.querySelector('#name');
+const messageData = document.querySelector('#message');
+const emptyObject = {};
+
+function storeData() {
+  emptyObject.namee = nameData.value;
+  emptyObject.email = emailE.value;
+  emptyObject.message = messageData.value;
+  localStorage.setItem('object', JSON.stringify(emptyObject));
+}
+
```

Comment:
- Hey Team, great job so far! There's a little issue that needs to be addressed with the form which is, after filling out the form, and not submitting it, the entered values should be saved in the browser when I reload the page. On the other hand, it works fine after submitting the form. Please, let's try fixing this to improve the user experience of our app.

Reply:
@zieeco 
The suggestion you made was not a requirement.
![image](https://user-images.githubusercontent.com/41428579/202652030-de35e59e-7b07-4853-9bcc-7aa181ac8e80.png)
Please check the video below demonstrating that I have met all the requirements above:
[Video Presentation](https://www.loom.com/share/e570671c23f14887bb228dd7468127d7)